package.json

```diff
@@ -129,17 +132,23 @@
     "eslint-plugin-filenames": "^1.3.2",
     "eslint-plugin-no-only-tests": "^2.3.1",
     "eslint-plugin-react-hooks": "^2.0.1",
-    "react-intl": "^2.4.0",
     "react": "^16.5.1",
     "react-dom": "^16.5.1",
+    "react-intl": "^2.4.0",
```

Comment:
react + intl dev depth? hm
not related to PR changes

Reply:
It looks like I've added new packages and they were re-arranged in alphabetical order

Reply:
yep, but why is react in devDepth?  better to ask Arvind I suppose...

Reply:
I've moved it to deps